client/pom.xml

```diff
@@ -132,7 +132,12 @@
             <groupId>org.yaml</groupId>
             <artifactId>snakeyaml</artifactId>
         </dependency>
-    
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-library</artifactId>
```

Comment:
Why add this dependency?

Reply:
I used some Matcher class in this library, maybe it is not necessary here. 
I have remove it.